src/main/java/agh/edu/zeuspol/services/HermesService.java

```diff
@@ -0,0 +1,67 @@
+package agh.edu.zeuspol.services;
+
+import agh.edu.zeuspol.datastructures.Rule;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class HermesService {
+  private final RestTemplate restTemplate;
+
+  //    @Value("${hermes.url}")
+  private String hermesUrl = "http://127.0.0.2:8080";
```

Comment:
please uncomment this value, you can set the default value in application.properties like this `hermes.url=${HERMES_URL:127.0.0.2}` if the environmental variable won't be foung, the default ( 127.0.0.2 ) will be used